app/src/examples/AnimatedSensorRotationExample.tsx

```diff
@@ -0,0 +1,45 @@
+import Animated, {
+  useAnimatedStyle,
+  useAnimatedSensor,
+  SensorType,
+} from 'react-native-reanimated';
+import { StyleSheet, View } from 'react-native';
+
+import React from 'react';
+
+export default function AnimatedSensorRotationExample() {
+  const gravity = useAnimatedSensor(SensorType.ROTATION);
+
+  const animatedStyle = useAnimatedStyle(() => {
+    const { pitch, roll, yaw } = gravity.sensor.value;
+    return {
+      transform: [
+        { rotateX: `${pitch}rad` },
+        { rotateY: `${roll}rad` },
+        { rotateZ: `${yaw}rad` },
```

Comment:
Could you please add a comment here with a link (preferably to some official iOS/Android docs) that proves that this mapping is indeed correct?